lib/rules/prefer-stateless-function/index.js

```diff
@@ -221,8 +223,16 @@ module.exports = {
           property.kind === 'constructor' &&
           isRedundantSuperCall(property.value.body.body, property.value.params)
         ;
+        var isStatic = property.static;
         var isRender = name === 'render';
-        return !isDisplayName && !isPropTypes && !contextTypes && !isUselessConstructor && !isRender;
+        return !(
```

Comment:
Agree ;) I've saved couple characters, but the readability of previous one is much better.